mosaic-runtime/src/commonTest/kotlin/com/jakewharton/mosaic/layout/OffsetTest.kt

```diff
@@ -41,7 +41,8 @@ class OffsetTest {
 					TestFiller(modifier = Modifier.size(1))
 				}
 			}
-			assertFails {
+			// check when going beyond the border of the TextSurface#get
```

Comment:
We should probably change the logic inside `TextSurface#get`, because otherwise using Modifier.offset is dangerous due to crashes, which is not present in the same cases in the Compose UI, for example. 
We can do as in https://github.com/JakeWharton/mosaic/pull/255 or in another way.